src/lib/isMACAddress.js

```diff
@@ -1,19 +1,18 @@
 import assertString from './util/assertString';
 
-const macAddress = /^([0-9a-fA-F][0-9a-fA-F]:){5}([0-9a-fA-F][0-9a-fA-F])$/;
-const macAddressNoColons = /^([0-9a-fA-F]){12}$/;
-const macAddressWithHyphen = /^([0-9a-fA-F][0-9a-fA-F]-){5}([0-9a-fA-F][0-9a-fA-F])$/;
-const macAddressWithSpaces = /^([0-9a-fA-F][0-9a-fA-F]\s){5}([0-9a-fA-F][0-9a-fA-F])$/;
-const macAddressWithDots = /^([0-9a-fA-F]{4}).([0-9a-fA-F]{4}).([0-9a-fA-F]{4})$/;
+const macAddress = /^(?:[0-9a-fA-F]{2}([-:\s]))([0-9a-fA-F]{2}\1){4}([0-9a-fA-F]{2})$/;
```

Comment:
@tux-tn Colons and whitespaces are still supported, by I joined the three cases in one regex. It should be less expensive in terms of performance.

Reply:
Sorry i didn't check the regex itself, that's good  👍